packages/ag-charts-website/src/content/docs/error-series/index.mdoc

```diff
@@ -1,5 +1,37 @@
 ---
 title: 'Error Series'
+enterprise: true
 ---
 
-TODO
+Error bars show the uncertainty of continuous data values. They can be enabled
```

Comment:
Should error bars be title cased ("Error Bars") throughout the documentation?

Reply:
Correct

Reply:
Done.